ArcGISRuntimeSDKQt_CppSamples/Layers/ListKmlContents/ListKmlContents.h

```diff
@@ -74,7 +73,7 @@ class ListKmlContents : public QObject
   QStringList m_levelNodeNames = {};
   QList<Esri::ArcGISRuntime::KmlNode*> m_kmlNodesList = {};
   Esri::ArcGISRuntime::KmlNode* m_currentNode = nullptr;
-  QString m_labelText = "";
+  bool m_lastLevel;
```

Comment:
I think functions like `displayPreviousLevel()` shout be setting this but don't. I'd suggest making a `bool hasDescendents(KmlNode* node)` function instead to remove the need for caching a value.